osgp/protocol-adapter-iec60870/osgp-protocol-simulator-iec60870/pom.xml

```diff
@@ -0,0 +1,164 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 Smart Society Services B.V. Licensed under the Apache License, 
+  Version 2.0 (the "License"); you may not use this file except in compliance with 
+  the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0 -->
+<project
+  xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>2.1.3.RELEASE</version>
+    <relativePath /> <!-- lookup parent from repository -->
+  </parent>
+  
+  <groupId>org.opensmartgridplatform</groupId>
+  <artifactId>osgp-protocol-simulator-iec60870</artifactId>
+  <version>4.35.0-SNAPSHOT</version>
+  <packaging>war</packaging>
+  <name>osgp-protocol-simulator-iec60870</name>
+  <description>Simulator for IEC60870-5-104 protocol</description>
+  <organization>
+    <name>OSGP</name>
+    <url>http://opensmartgridplatform.org</url>
+  </organization>
+  <licenses>
+    <license>
+      <name>APACHE 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.html</url>
+    </license>
+  </licenses>
+  <!-- URL is passed in as a maven argument: -Dmaven.site.url=http://my/url/for/the/site -->
+  <url>${maven.site.url}</url>
+  <distributionManagement>
+    <site>
+      <id>pa-iec60870-simulator</id>
+      <name>pa-iec60870-simulator</name>
+      <!-- URL is passed in as a maven argument: -Dmaven.site.distributionManagement.site.url=file:////the/path/to/deploy/the/site/to -->
+      <url>${maven.site.distributionManagement.site.url}</url>
+    </site>
+  </distributionManagement>
+  
+  <properties>
+    <display.version>${project.version}-${BUILD_TAG}</display.version>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <java.version>1.8</java.version>
+    <openmuc.j60870.version>1.2.1</openmuc.j60870.version>
+    
+    <jxr.version>2.5</jxr.version>
+    <maven.project.info.reports.plugin.version>3.0.0</maven.project.info.reports.plugin.version>
+    <maven.site.plugin>3.7.1</maven.site.plugin>
+    
+  </properties>
+
+  <dependencies>
+    <!-- j60870 -->
+    <dependency>
+      <groupId>org.openmuc</groupId>
+      <artifactId>j60870</artifactId>
+      <version>${openmuc.j60870.version}</version>
+    </dependency>  
+
+    <!--  JUnit 5 dependencies -->
+    <dependency>
+        <groupId>org.junit.jupiter</groupId>
+        <artifactId>junit-jupiter-api</artifactId>
+        <scope>test</scope>
+    </dependency>
+    <dependency>
+        <groupId>org.junit.jupiter</groupId>
+        <artifactId>junit-jupiter-engine</artifactId>
+        <scope>test</scope>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this line start at column 7. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=xml%3AIndentCheck)